chunky/src/res/se/llbit/chunky/ui/render/GeneralTab.fxml

```diff
@@ -1,18 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.geometry.Insets?>
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.CheckBox?>
-<?import javafx.scene.control.ComboBox?>
-<?import javafx.scene.control.Label?>
-<?import javafx.scene.control.Separator?>
-<?import javafx.scene.layout.HBox?>
-<?import javafx.scene.layout.VBox?>
-<?import javafx.scene.control.ScrollPane?>
-<?import se.llbit.chunky.ui.IntegerAdjuster?>
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import se.llbit.chunky.ui.*?>
 
-<?import javafx.scene.layout.FlowPane?>
-<fx:root type="javafx.scene.control.ScrollPane" xmlns="http://javafx.com/javafx/8.0.40" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root type="javafx.scene.control.ScrollPane" xmlns="http://javafx.com/javafx/11.0.10-internal" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
I have no idea if/how the namespace is used by javafx but checked that this still works with java 8 give that you've replaced 8.something with 11.something. (or maybe we don't care because soon java will be bundled?)

Reply:
I'll revert these namespaces, just in case.

Reply:
A name space that ends with -internal doesn't sound like we should use it regardless of the Java version 🤔